src/helpers/responsiveness.ts

```diff
@@ -8,19 +8,22 @@ const maxMedia: Media = {
   phone: interpolate('phone', 'max'),
   tablet: interpolate('tablet', 'max'),
   desktop: interpolate('desktop', 'max'),
+  largeDesktop: interpolate('largeDesktop', 'max'),
```

Comment:
We should start a migration guide so we don't forget to include any specific content